sp/src/game/server/physics_prop_ragdoll.cpp

```diff
@@ -58,6 +60,8 @@ const float ATTACHED_DAMPING_SCALE = 50.0f;
 #define	SF_RAGDOLLPROP_STARTASLEEP			0x10000
 #ifdef MAPBASE
 #define	SF_RAGDOLLPROP_FIXED_CONSTRAINTS	0x20000
+#define	SF_RAGDOLLPROP_ALLOW_USE			0x40000
```

Comment:
Are you sure you want to add new spawnflags? The likelihood of Mapbase mods adding their own is low, but if they do then this update will be breaking.